django_crontab/crontab.py

```diff
@@ -14,7 +14,7 @@
 
 from django_crontab.app_settings import Settings
 
-string_type = basestring if sys.version_info[0] == 2 else str  # flake8: noqa
+string_type = basestring if sys.version_info[0] == 2 else str  # noqa: F821
```

Comment:
Not quite sure why this was now failing, as it's got nothing to do with my changes. Changing to an explicit ignore seems to work fine